### PR TITLE
[ZEPPELIN-407] Improve document on how to manage external libraries in spark interpreter

### PR DESCRIPTION
**Pull request by minahlee**
Improve docs on library loading in spark interpreter.
* Add information to load libraries using spark properties besides dependency loading using %dep interpreter
* State different ways of dependency loading for different version of Zeppelin
* Move Spark specific information from install.md to spark.md

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Great work.
Looks good to me.

File: docs/docs/interpreter/spark.md [hunk 1]
```diff
@@ -103,6 +106,58 @@ z.load("groupId:artifactId:version").local()
...
+    <td>Comma-separated list of maven coordinates of jars to include on the driver and executor classpaths. Will search the local maven repo, then maven central and any additional remote repositories given by --repositories. The format for the coordinates should be groupId:artifactId:version.</td>
+  </tr>
+  <tr>
+    <td>spark.files</td>
+    <td>--files</td>
+    <td>%pyspark</td>
+    <td>Comma-separated list of files to be placed in the working directory of each executor.</td>
+  </tr>
+</table>
+Note that adding jar to pyspark is only availabe via %dep interpreter at the moment
```
Comment (felixcheung) [inline docs/docs/interpreter/spark.md]: Could you perhaps put this line at the beginning of the section?

File: docs/docs/interpreter/spark.md [hunk 2]
```diff
@@ -103,6 +106,58 @@ z.load("groupId:artifactId:version").local()
 
 Note that %dep interpreter should be used before %spark, %pyspark, %sql.
```
Comment (felixcheung) [inline docs/docs/interpreter/spark.md]: Could you perhaps put this line at the beginning of the section?

**Comment by felixcheung** [comment]
looks good, this covers most of this earlier bug: ZEPPELIN-368
I'd say we would need to add: complete list of properties supported by interpreters (eg. there're a few for %dep), using z to pass DataFrame, add more on starting Zeppelin when SPARK_HOME is set (provided Spark) (or rather, how to config Spark if SPARK_HOME is not set?)

**Comment by minahlee** [comment]
@felixcheung I'd like to help [ZEPPELIN-368](https://issues.apache.org/jira/browse/ZEPPELIN-368?jql=project%20%3D%20ZEPPELIN). Do you mind if I create another PR for this?

**Comment by felixcheung** [comment]
Sure! Please feel free.

**Comment by Leemoonsoo** [comment]
While this PR answers popular questions "How to load custom jar?" well, the other improvement on doc can be handled separately after merging this PR.
Looks good to me. merge if there're no more discussions.

**Pull request closed without merging**